Bottom: 48f56b1ae7023d66928f21329d13d27c51de1707
Top:    71ef58a3dc2fb4dd2a392e7f4587d76e09e077ca
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 23:48:46 +0000

hid/gerber: Remove unused variable outline_trace_size

Left over accidentally from commit 35b137c3e193471e07f0873dfb149cebd059dca2


---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index b7e99bb..27496bf 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -93,7 +93,6 @@ static int is_drill;
 static int current_mask;
 static int flash_drills;
 static int copy_outline_mode;
-static int outline_trace_size;
 static LayerTypePtr outline_layer;
 
 enum ApertureShape
